app/src/main/java/com/wireguard/android/fragment/TunnelListFragment.kt

```diff
@@ -333,12 +333,12 @@ class TunnelListFragment : BaseFragment() {
         }
         binding!!.fragment = this
         Application.tunnelManager.getTunnels().thenAccept { binding!!.tunnels = it }
-        binding!!.rowConfigurationHandler =
-            ObservableKeyedRecyclerViewAdapter.RowConfigurationHandler { binding: TunnelListItemBinding, tunnel: Tunnel, position ->
-                binding.fragment = this
+        binding!!.rowConfigurationHandler = object : ObservableKeyedRecyclerViewAdapter.RowConfigurationHandler<TunnelListItemBinding,Tunnel>{
+            override fun onConfigureRow(binding: TunnelListItemBinding, item:Tunnel, position: Int) {
```

Comment:
Can the `Tunnel` type parameter contine being called `tunnel` here?

Reply:
For ' !! ' . Application.backend.enumerate() function can return null as per it's definition. In Kotlin , nullabe checks are there . I didn't thought it's best to report for null here.